doc/guide/cockpit-channel.xml

```diff
@@ -142,7 +142,7 @@ channel.close([options])
   <refsection id="cockpit-channels-message">
     <title>channel.onmessage</title>
 <programlisting>
-$(channel).on("message", function(event, data) { ... })
+channel.addEventListener("message", function(event, data) { ... })
 channel.addEventListener("message", function(event, data) { ... })
```

Comment:
The `addEventListener` line is duplicated now. I think we had both the jQuery and pure DOM version in here before.

There's a few more instances of this in this file.